pulpcore/cli/deb/distribution.py

```diff
@@ -32,6 +37,12 @@
         " Specified as '[[<plugin>:]<type>:]<name>' or as href."
     ),
 )
+checkpoint_option = pulp_option(
+    "--checkpoint/--not-checkpoint",
```

Comment:
```suggestion
    "--checkpoint/--no-checkpoint",
```
Not sure how this compares with other plugins, but so far all the toggle flags in pulp-cli-deb use `--no-` not `--not-`.

Reply:
For rpm, I used `--not-checkpoint` and grammatically, I think `not` makes more sense. But if we want consistency with other toggle flags, I am perfectly happy to use `no`.

Reply:
Looking at the following in Pulp CLI:
```
$ git grep '\-\-not\-' pulpcore/
pulpcore/cli/core/worker.py:    click.option("--missing/--not-missing", default=None),
pulpcore/cli/core/worker.py:    click.option("--online/--not-online", default=None),
pulpcore/cli/file/distribution.py:        "--checkpoint/--not-checkpoint",
pulpcore/cli/rpm/distribution.py:        "--checkpoint/--not-checkpoint",
```
We can say:

- Both file and rpm have `--not-checkpoint`.
- There are other flags like `--not-missing` where `--no-missing` really does not make sense.
- We will not have all flags use `--no-` for the `false` option.

=> We should keep it `--not-checkpoint` to keep it consistent with the other plugins.